src/createDelete.js

```diff
@@ -1,25 +1,29 @@
-import dots from './assets/dots.png';
-import deleteIcon from './assets/delete.png';
 import { tasks } from './addTask.js';
 import * as storage from './storageFunc.js';
+import deleteIcon from './assets/delete.png';
+import dots from './assets/dots.png';
 
 const list = document.querySelector('.list');
 
-const createTask = () => {
+const createTask = (loadedTasks) => {
+  const taskList = loadedTasks || tasks;
+
   list.innerHTML = '';
-  tasks.forEach((task, index) => {
+
+  taskList.forEach((task, index) => {
```

Comment:
- [ ] Hi, thank you for submitting your code for review. I appreciate your effort and creativity in creating the task list functionality. However it is a project requirement to export methods related to status updates (`completed`: `true` / `false`) of the tasks. A better practice would be to create a separate file for this method and import it as a modules. This way, you can keep your code organized and reusable, and avoid repeating yourself. For example, you could create a file called status.js and export the following methods:


If you only need the method related to the status updates that involve changing the checkbox, you can create a file called checkbox.js and export the following method:

```javascript
// checkbox.js

// change the text decoration of the task and save the changes to the storage based on the checkbox status
export const toggleCheckbox = (taskText, task, checkbox) => {
  task.completed = checkbox.checked;
  taskText.style.textDecoration = checkbox.checked ? 'line-through' : 'none';
  storage.saveTasks(tasks);
};
```

Then, in your index.js file, you can import this method and use it in your event listener:

```javascript
// index.js

import { toggleCheckbox } from './checkbox.js';

// ...

checkbox.addEventListener('change', () => {
  toggleCheckbox(taskText, task, checkbox);
});
```

Please try to implement this solution and resubmit your code for review. If you have any questions or need further assistance, feel free to reach out to me. Happy coding! 😊